RENOVATE.md

```diff
@@ -0,0 +1,153 @@
+releases for a package will be separated into its own update.
+**[:combinePatchMinorReleases](https://docs.renovatebot.com/presets-default/#combinepatchminorreleases)** - Patch and
+minor releases for a single package will be combined to a single update.
+**[:enableVulnerabilityAlerts](https://docs.renovatebot.com/presets-default/#enablevulnerabilityalerts)** - Open a PR
+should the repository have any vulnerability alerts (see below).
+**[:timezone(UTC)](https://docs.renovatebot.com/presets-default/#timezonearg0)** - Not strictly necessary but ensures
+schedules use the UTC timezone.
+**[:label(renovate)](https://docs.renovatebot.com/presets-default/#labelarg0)** - Add the renovate label to any PRs.
+**[group:allNonMajor](https://docs.renovatebot.com/presets-group/#groupallnonmajor)** - Any non major updates will be
+grouped into a single update.
+**[schedule:daily](https://docs.renovatebot.com/presets-schedule/#scheduledaily)** - Run only once per day (but multiple
+times a run, until everything is updated/rebased).
+
+#### Vulnerability Alerts
+
+From https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts:
+
+> Renovate can read from GitHub's Vulnerability Alerts and customize Pull Requests accordingly. For this to work, you must first ensure you have enabled "Dependency graph" and "Dependabot alerts" under the "Security & analysis" section of the repository's "Settings" tab.
+>
+> Additionally, if you are running Renovate in app mode then you must make sure that the app has been granted the permissions to read "Vulnerability alerts". If you are the account admin, browse to the app (e.g. https://github.com/apps/renovate), select "Configure", and then scroll down to the "Permissions" section and verify that read access to "vulnerability alerts" is mentioned.
+>
+> Once the above conditions are met, and you have received one or more vulnerability alerts from GitHub for this repository, then Renovate will attempt to raise fix PRs accordingly.
+
+### Other Settings
+
+```json
+"enabledManagers": ["composer", "dockerfile", "docker-compose", "github-actions"],
+"lockFileMaintenance": {"enabled": true, "extends": ["schedule:daily"]},
+"platformAutomerge": true,
+"prFooter": "[Read more information](https://github.com/laminas/.github/blob/main/RENOVATE.md) about the use of [Renovate Bot](https://github.com/renovatebot/renovate) within Laminas.",
+"rangeStrategy": "replace",
+"rollbackPrs": true,
+```
+
+**[enabledManagers](https://docs.renovatebot.com/configuration-options/#enabledmanagers)** - To begin with, this is
+enabled for composer, docker and github-actions. Other managers (such as npm etc) are also available.
+**[lockFileMaintenance](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance)** - Lockfile
+maintenance should be performed during the scheduled run.
+**[platformAutomerge](https://docs.renovatebot.com/configuration-options/#platformautomerge)** - Use github's merging
```

Comment:
```suggestion
**[platformAutomerge](https://docs.renovatebot.com/configuration-options/#platformautomerge)** - Use GitHub's merging
```

GitHub is a proper noun, so should be capitalized.

Reply:
Thanks, committed.